### PR TITLE
Review

### PR DESCRIPTION
**Pull request by dincamihai**
Tracker issue: https://github.com/SUSE/spacewalk/issues/9819

### CODE REVIEW

File: modules/salt/pages/large-scale-tuning.adoc [hunk 1]
```diff
@@ -50,6 +50,7 @@ Tuning one parameter might require tuning other parameters, or changing hardware
 When you change a parameter, follow the arrows from that node on the graph to determine what other parameters might need adjustment.
 Continue through each parameter until you have visited all nodes on the graph.
 
+// missing file?
```
Comment (Loquacity) [inline modules/salt/pages/large-scale-tuning.adoc]: image

Nope. Definitely still there.

File: modules/salt/pages/salt-timeouts.adoc [hunk 2]
```diff
@@ -24,7 +24,7 @@ If some of the clients are down during this process, the workflow continues as f
 
 . If `timeout` is reached before getting all expected responses from the clients, then Salt master would trigger an aditional job (a Salt [command]``find_job`` job) targeting only pending clients to check whether the job is already running on the client.
 . Now `gather_job_timeout` is evaluated. A new counter is now triggered.
-. If this new [command]``find_job`` job responses that the original job is actually running on the client, then Salt master will wait for that client's response.
+. If this new [command]``find_job`` job responses that the original job is actually running on the client, then Salt master will wait for that client's response for another `gather_job_timeout` interval before issuing another [command]``find_job``
```
Comment (keichwa) [inline modules/salt/pages/salt-timeouts.adoc]:
```suggestion
. If this new [command]``find_job`` job responses that the original job is actually running on the client, then Salt master will wait for that client's response for another `gather_job_timeout` interval before issuing the next [command]``find_job`` job.
```

File: modules/salt/pages/large-scale-tuning.adoc (hunk 1 above)
Comment (dincamihai) [reply modules/salt/pages/large-scale-tuning.adoc]: Somehow the link is wrong: https://github.com/SUSE/doc-susemanager/blob/master/modules/salt/pages/parameter_graph.svg

Comment (Loquacity) [reply modules/salt/pages/large-scale-tuning.adoc]: no, it won't render in github.

File: modules/salt/pages/salt-timeouts.adoc [hunk 3]
```diff
@@ -24,7 +24,7 @@ If some of the clients are down during this process, the workflow continues as f
 
 . If `timeout` is reached before getting all expected responses from the clients, then Salt master would trigger an aditional job (a Salt [command]``find_job`` job) targeting only pending clients to check whether the job is already running on the client.
 . Now `gather_job_timeout` is evaluated. A new counter is now triggered.
-. If this new [command]``find_job`` job responses that the original job is actually running on the client, then Salt master will wait for that client's response.
+. If this new [command]``find_job`` job responses that the original job is actually running on the client, then Salt master will wait for that client's response for another `gather_job_timeout` interval before issuing the next [command]``find_job`` job.
```
Comment (Loquacity) [inline modules/salt/pages/salt-timeouts.adoc]:
```suggestion
. If this new [command]``find_job`` job responds that the original job is actually running on the client, then Salt master will wait for that client's response for another `gather_job_timeout` interval before issuing the next [command]``find_job`` job.
```

Comment (Loquacity) [reply modules/salt/pages/salt-timeouts.adoc]: found another one ;)

**Comment by dincamihai** [comment]
I'm not sure if I'm allowed to merge it myself. Please feel free to merge it if there's nothing left here.
Thanks for reviewing the review :)

**Pull request merged**